adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/impl/LoraProtocolAdapter.java

```diff
@@ -334,7 +334,8 @@ private void scheduleStartLoraCommandConsumer(final String tenantId) {
         return getCommandConsumerFactory().createCommandConsumer(
                 tenantId,
                 LORA_COMMAND_CONSUMER_DEVICE_ID,
-                receivedCommandContext -> commandConsumer(tenantId, receivedCommandContext));
+                receivedCommandContext -> commandConsumer(tenantId, receivedCommandContext),
+                null);
```

Comment:
As the method later on calls `span.context()` I would expect the method to fail. Why not pass `NoopSpan.INSTANCE`?

Reply:
The `null` stands for the `SpanContext` parameter here, and that may be `null`. So I don't think it is a problem.

Reply:
Indeed, it looked to me like this was the `Span` parameter.